cmd/create/cluster/cmd.go

```diff
@@ -862,6 +881,50 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	enableCustomerManagedKey := args.enableCustomerManagedKey
+
+	if interactive.Enabled() {
+		enableCustomerManagedKey, err = interactive.GetBool(interactive.Input{
+			Question: "Enable Customer Managed key",
+			Help:     cmd.Flags().Lookup("enable-customer-managed-key").Usage,
+			Default:  enableCustomerManagedKey,
+			Required: false,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid value for enable-customer-managed-key: %s", err)
+			os.Exit(1)
+		}
+	}
+	kmsKeyARN := args.kmsKeyARN
+
+	if !enableCustomerManagedKey && kmsKeyARN != "" {
+		reporter.Errorf("KMS Key ARN can be specified only if the customer managed key is enabled")
+		os.Exit(1)
+	}
+
+	if enableCustomerManagedKey && interactive.Enabled() {
```

Comment:
We should drop the user in interactive mode if they enable the managed key without providing it:
```
if enableCustomerManagedKey && kmsKeyARN == "" {
  interactive.Enable()
}
```

Reply:
That means that this line should really be `||` instead of `&&`
```
if enableCustomerManagedKey || interactive.Enabled() {
```